### PR TITLE
Add process method

### PR DESCRIPTION
**Pull request by Iva-khar**
Added process method to Converter class, made models folder for different models

### CODE REVIEW

File: ratu/services.py [hunk 1]
```diff
@@ -52,11 +55,101 @@ def parse_file(self):
...
+                    name=record['OBL_NAME']
+                    )
+                region.save()
+                region_list.insert(0, record['OBL_NAME'])
+                region=Region.objects.get(name=record['OBL_NAME'])
+
+            if record['REGION_NAME']:
+                a=record['REGION_NAME']
+            else:
+                a=999
```
Comment (criticus) [inline ratu/services.py]: add a comment to explain why it's 999 or better make a const

File: ratu/services.py [hunk 2]
```diff
@@ -52,11 +55,101 @@ def parse_file(self):
...
+                    name=a
+                    )
+                district.save()
+                district_list.insert(0, [region, a])
+                district=District.objects.get(name=a, region=district.region)
+
+            if record['CITY_NAME']:
+                b=record['CITY_NAME']
+            else:
+                b=999
```
Comment (criticus) [inline ratu/services.py]: same here

File: ratu/services.py [hunk 3]
```diff
@@ -52,11 +55,101 @@ def parse_file(self):
...
+                    name=b
+                    )
+                city.save()
+                city_list.insert(0, [region, district, b])
+                city=City.objects.get(name=b, region=district.region, district=city.district)
+
+            if record['CITY_REGION_NAME']:
+                c=record['CITY_REGION_NAME']
+            else:
+                c=999
```
Comment (criticus) [inline ratu/services.py]: same here

**Comment by criticus** [comment]
and remove .xml files we do not need them in the PR

File: ratu/services.py (hunk 1 above)
Comment (Iva-khar) [reply ratu/services.py]: done

File: ratu/services.py (hunk 2 above)
Comment (Iva-khar) [reply ratu/services.py]: done

File: ratu/services.py (hunk 1 above)
Comment (Iva-khar) [reply ratu/services.py]: Add constant to ratu_models

File: ratu/services.py (hunk 3 above)
Comment (Iva-khar) [reply ratu/services.py]: Add constant to ratu_models

File: ratu/services.py [hunk 4]
```diff
@@ -52,11 +55,103 @@ def parse_file(self):
...
+        with codecs.open(self.LOCAL_FOLDER + self.LOCAL_FILE_NAME, encoding="cp1251") as file:
+            data = xmltodict.parse(file.read())
+        
+        #creating lists for registration items that had writed to db 
+        region_list = []
+        district_list = list()
+        city_list = list()
+        citydistrict_list = list()
+
+        EMPTY_FIELD = 'empty field'
```
Comment (criticus) [inline ratu/services.py]: if you added it to the model why it's here?

Comment (Iva-khar) [reply ratu/services.py]: one moment...

Comment (Iva-khar) [reply ratu/services.py]: deleted from process

**Pull request closed without merging**